packages/material-ui/src/ListItem/ListItem.test.js

```diff
@@ -72,71 +80,77 @@ describe('<ListItem />', () => {
           </ListContext.Consumer>
         </ListItem>,
       );
-      assert.strictEqual(context.dense, false);
-      wrapper.setProps({ dense: true });
-      assert.strictEqual(context.dense, true);
+      expect(context).to.have.property('dense', false);
+      setProps({ dense: true });
+      expect(context).to.have.property('dense', true);
     });
   });
 
   describe('secondary action', () => {
     it('should wrap with a container', () => {
-      const wrapper = mount(
+      const { getByRole } = render(
         <ListItem>
           <ListItemText primary="primary" />
           <ListItemSecondaryAction />
         </ListItem>,
       );
-      const listItem = findOutermostIntrinsic(wrapper);
-      assert.strictEqual(listItem.hasClass(classes.container), true);
-      assert.strictEqual(wrapper.find('li > div').exists(), true);
```

Comment:
This didn't do what it was intended to do since it also matched `<li><span /><div /></li>` which is important for the next test.